glfw.go

```diff
@@ -53,8 +53,6 @@ func (m *windowManager) sendPointerEvent(window *glfw.Window, phase embedder.Poi
 		return
 	}
 
-	// TODO(GeertJohan): sometimes the x and/or y given by glfw is negative or over window size, could this cause an issue?
```

Comment:
x and/or y given by glfw can be negative, this occurs when the cursor is dragged (kMove) away from the window.